webapp/src/ts/modals/edit-message-group/edit-message-group.component.html

```diff
@@ -0,0 +1,37 @@
+<mm-modal
+  [status]="status"
+  id="edit-message-group"
+  [titleKey]="'Edit'"
+  [submitKey]="'save'"
+  [submittingKey]="'saving'"
+  (onCancel)="cancel()"
+  (onSubmit)="submit()"
+>
+  <form action="" method="POST">
+
+    <fieldset *ngFor="let task of model.group.rows; index as index" [disabled]="task.state !== 'scheduled' && task.state !== 'muted'">
```

Comment:
Checking the state data directly here (and on line 17) means we have to take on more logic here (and anywhere else that may need a similar check). To me it looks like tasks might have a concept of "disabled" or "deletable". If that's the case, the task object could have methods like `isMuted` and `isDeletable` to be called in these places. If it's the case that such logic is more about the view than what a `task` is about, a [decorator](https://refactoring.guru/design-patterns/decorator) might be an appropriate place to put those methods, but that also may be getting outside the scope of the migration.

Reply:
Tasks have states. https://docs.communityhealthtoolkit.org/apps/guides/messaging/sms-states/#message-states-in-medic
States like "scheduled" and "muted" means are "deletable". And you can also "mute" a scheduled task or "schedule" a muted task. 

I purposefully didn't change this, and very purposefully wanted to get these tasks from redux `as-is`. They're dumb objects, pieces of a report's json body, and I didn't want to overcomplicate right now.